src/arduino/arduino.ts

```diff
@@ -725,12 +725,11 @@ export class ArduinoApp {
             }
             return ret;
         }
-        let stdoutbuf = ""
+        let stdoutbuf = "";
         const stdoutcb = (line: string) => {
             if (cocopa.callback) {
-                stdoutbuf += line
-                let lines = [stdoutbuf];
-                lines = stdoutbuf.split('\n');
+                stdoutbuf += line;
+                let lines = stdoutbuf.split('\n');
                 stdoutbuf = lines.pop()!;

```

Comment:
This statement can be removed, there is no point in updating `stdoutbuf` after its last usage.

Reply:
The stdoutbuf is upvalue variable of the closure, the last update is taking the remain characters that in last line.